packages/account-v2/src/components/FormFields/FormInputField.tsx

```diff
@@ -17,19 +20,25 @@ type FormInputFieldProps = Omit<ComponentProps<typeof Input>, 'errorMessage' | '
  * @param [props] - Other props to pass to Input
  * @returns ReactNode
  */
-const FormInputField = ({ name, validationSchema, ...rest }: FormInputFieldProps) => (
-    <Field name={name} validate={validateField(validationSchema)}>
-        {({ field, meta: { error, touched } }: FieldProps<string>) => (
-            <Input
-                {...field}
-                {...rest}
-                aria-label={rest.label}
-                autoComplete='off'
-                error={Boolean(error && touched)}
-                message={touched && error}
-            />
-        )}
-    </Field>
-);
+export const FormInputField = ({ name, validationSchema, ...rest }: FormInputFieldProps) => {
+    const formik = useFormikContext();
 
-export default FormInputField;
+    if (!formik) {
+        throw new Error('FormInputField must be used within a Formik component');
+    }
+
+    return (
+        <Field name={name} validate={validateField(validationSchema)}>
+            {({ field, meta: { error, touched } }: FieldProps<string>) => (
+                <Input
+                    {...field}
+                    {...rest}
+                    aria-label={rest.label}
+                    autoComplete='off'
+                    error={Boolean(error && touched)}
+                    message={error && touched ? error : rest.message}
```

Comment:
do we need parenthesis here to make it more understandable?
you may ignore this as well

Reply:
It is getting removed by prettier 😕